types/underscore/underscore-tests.ts

```diff
@@ -575,12 +587,20 @@ _(explicitNumberDictionary).each((value, key, collection) => {
     _.chain(null).get(["a"], numberValue);
 
     // chained without default value
-    // $ExpectType _Chain<string | number | undefined, string | number | undefined>
+    // $ExpectType _Chain<undefined, number | undefined>
     _.chain({ a: numberValue, b: stringValue }).get(["a"]);
 
     // chained with default value
-    // $ExpectType _Chain<string | number, string | number>
+    // $ExpectType _Chain<string, string | number>
```

Comment:
These changes confuse me. Why is `number` no longer expected for the `T` parameter of the new chain?

Reply:
Sorry, that was my mistake. I fixed it to be `_Chain<never, string | number>` since `_.chain(stringValue)` is `_Chain<never, string>`.

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67318/commits/09924cb0cb4456c7ea0a2f8fe2dc8641a128a745

Reply:
`_.chain(obj).get(path)` is indeed equivalent to `_.chain(_.get(obj, path))`. So I guess your typings are right. It seems strange, but I must confess that I never fully understood the `_Chain` typing.

I will submit a review with an approval after you've addressed the CI failure.